### PR TITLE
pw-mon: add page and pt_BR translation

### PR DESCRIPTION
**Pull request by vitorhcl**
- [X] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [X] The page(s) have at most 8 examples.
- [X] The page description(s) have links to documentation or a homepage.
- [X] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [X] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Review by gutjuri — APPROVED** [review]
Thanks!

File: pages/linux/pw-mon.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+- Monitor a specific remote instance:
+
+`pw-mon --remote={{remote_name}}`
+
+- Monitor the default instance specifying a color configuration:
+
+`pw-mon --color={{never|always|auto}}`
+
+- Show help:
```
Comment (kbdharun) [inline pages/linux/pw-mon.md:18]:
```suggestion
- Display help:
```

We generally go with this generic term across our pages.

**Review by kbdharun — APPROVED** [review]
LGTM, Welcome to tldr and thanks for your contribution.

**Review by isaacvicente — APPROVED** [review]
LGTM :tada:

**Comment by isaacvicente** [comment]
@kbdharun Shouldn't be @vitorhcl a reviewer of pt_BR translations?

**Comment by kbdharun** [comment]
> @kbdharun Shouldn't be @vitorhcl a reviewer of pt_BR translations?

Agreed, I would love to have them onboard too, I will open an issue after 1 more PR is merged (currently they have 4 PRs merged).

**Pull request merged**